fapolicy_analyzer/ui/strings.py

```diff
@@ -118,3 +118,24 @@
 GET_GROUPS_LOG_ERROR_MSG = _(
     "An error occurred trying to retrieve the group list. Please try again."
 )
+
+TRUST_DB_READ_FAILURE_DIALOG_TITLE = _("Trust Database Read Failure")
+TRUST_DB_READ_FAILURE_DIALOG_TEXT = _(
+    """
+              The fapolicyd trusted resources database
```

Comment:
I'd just say "trust database" here

Reply:
Title text change to "Trust Database." Done.